<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesStylesEditor</class>
 <widget class="QDialog" name="NotesStylesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Notes Styles Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="NSlistBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="NewButton">
         <property name="text">
          <string>Add New Style</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="NewNameLabel">
         <property name="text">
          <string>New Style Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="NewNameEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="FootRadio">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Footnotes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="EndRadio">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Endnotes</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="NumberingLabel">
         <property name="text">
          <string>Numbering</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="NumberingBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="RangeLabel">
         <property name="text">
          <string>Range</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="RangeBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="StartLabel">
         <property name="text">
          <string>Start Number</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="StartSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="PrefixLabel">
         <property name="text">
          <string>Prefix</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="PrefixEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="SuffixLabel">
         <property name="text">
          <string>Suffix</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="SuffixEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="SuperMasterCheck">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="SuperMasterLabel">
         <property name="text">
          <string>Superscript in text</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="SuperNoteCheck">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="SuperNoteLabel">
         <property name="text">
          <string>Superscript in notes</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="charStyleComboLabel">
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string>Mark Character Style</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="CharStyleComboBox" name="charStyleCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="paraStyleComboLabel">
         <property name="text">
          <string>Note Paragraph Style</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="ParaStyleComboBox" name="paraStyleCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="AutoH">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="AutoHLabel">
         <property name="text">
          <string>Auto Height</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="AutoW">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="AutoWLabel">
         <property name="text">
          <string>Auto Width</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QCheckBox" name="AutoWeld">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="AutoWeldLabel">
         <property name="text">
          <string>Auto Welding</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QCheckBox" name="AutoRemove">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="AutoRemoveLabel">
         <property name="text">
          <string>Remove if empty</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ApplyButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OKButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CharStyleComboBox</class>
   <extends>QComboBox</extends>
   <header>ui/spalette.h</header>
  </customwidget>
  <customwidget>
   <class>ParaStyleComboBox</class>
   <extends>QComboBox</extends>
   <header>ui/spalette.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
